app/controllers/reorder_timeline_entries_controller.rb

```diff
@@ -0,0 +1,21 @@
+class ReorderTimelineEntriesController < ApplicationController
+  before_action :require_unreleased_feature_permissions!
+  before_action :require_coronavirus_editor_permissions!
+  layout "admin_layout"
+
+  def index
+    coronavirus_page
+  end
+
+  def update; end
+
+private
+
+  def slug
+    params[:slug] || params[:coronavirus_page_slug]
```

Comment:
Hmmm, you're right. Also why did I decide to query ModelBuilder rather than get the actual CoronavirusPage. I think you've spotted some egregious copy and pasting.